.github/workflows/main.yml

```diff
@@ -129,7 +131,7 @@ jobs:
         run:  |
           # The generated firmware binary looks like "pinetime-mcuboot-app-0.8.2.bin"
           ls -l build/src/pinetime-mcuboot-app*.bin
-          ${{ runner.temp }}/mcuboot/scripts/imgtool.py create --align 4 --version 1.0.0 --header-size 32 --slot-size 475136 --pad-header build/src/pinetime-mcuboot-app*.bin build/src/pinetime-mcuboot-app-img.bin
+          ${{ runner.temp }}/mcuboot/scripts/imgtool.py create --align 4 --version 1.0.0 --header-size 32 --slot-size 475136 --pad-header build/src/pinetime-mcuboot-app-[0-9]*.bin build/src/pinetime-mcuboot-app-img.bin
```

Comment:
This fixes:
![image](https://user-images.githubusercontent.com/5289520/127422604-ac95c19a-342b-42b1-81fb-62016f4ae2e8.png)

The new glob pattern excludes `pinetime-mcuboot-app-image-1.3.0.bin` so there is only one input file to pass to the `create` command.